src/test/resources/Features/OCD-1566-updateDefaultSearchParametersForRecentlyRetiredACBs.feature

```diff
@@ -0,0 +1,45 @@
+@Regression
+Feature: OCD-1566 - Update default search parameters for recently retired ACBs
+  This test checks whether CCHIT is unchecked on Search page and ACBs can be retired and unretired 
+
+  Scenario: Verify CCHIT checkbox is unchecked on Search page
+    Given I am on CHPL search page
+    And I look at ACB filter options selected for default search
+    Then I see that "CCHIT" checkbox is unchecked
+
+  Scenario Outline: Verify ACBs can be retired
+    Given I'm logged in as "ROLE_ADMIN"
+    When I navigate to ONC ACB Management page
+    And I click on "<ACB Name>"
+    And I edit an ACB 
+    And I mark it as retired on "08/01/2015"
+    Then ONC ACB have retired status as "Yes"
+    Examples:
+      |ACB Name      |
+      |InfoGard      |
+      |ICSA Labs     |
+      |Drummond Group|
+      |SLI Compliance|
+
+  Scenario Outline: Verify ACBs can be unretired
+    Given I'm logged in as "ROLE_ADMIN"
+    When I navigate to ONC ACB Management page
+    And I click on "<ACB Name>"
+    And I edit an ACB
+    And I unretire an existing retired ACB
+    Then ONC ACB have retired status as "No"
+    Examples:
+      |ACB Name      |
+      |InfoGard      |
+      |ICSA Labs     |
+      |Drummond Group|
+      |SLI Compliance|
+
+  Scenario: Verify ACBs display an error message if any future date of retirement is selected 
+    Given I'm logged in as "ROLE_ADMIN"
+    When I navigate to ONC ACB Management page
+    And I click on "InfoGard"
+    And I edit an ACB 
+    And I mark it as retired on "08/01/2020"
```

Comment:
What happens on August 2nd of 2020? I know that's a long time from now, but we want our system to be future proof. Maybe set the year to a hundred years from now, or maybe set the date to "now + 1 day", but either way, a specific date will fail before we're ready for it

Reply:
I have just selected a future date that throws an error "Retirement date is required and must be before "now". 
I will set the year to a hundred years from now.